projects/swimlane/ngx-charts/src/lib/bar-chart/series-horizontal.component.ts

```diff
@@ -252,11 +254,13 @@ export class SeriesHorizontal implements OnChanges {
     this.tooltipType = this.tooltipDisabled ? undefined : StyleTypes.tooltip;
   }
 
-  isActive(entry: Series): boolean {
+  isActive(entry: DataItem): boolean {
     if (!this.activeEntries) return false;
-    const item = this.activeEntries.find(d => {
-      return entry.name === d.name && entry.series === d.series;
+
+    const item = this.activeEntries.find(active => {
+      return entry.name === active.name && entry.value === active.value;
```

Comment:
I think here the typings were off (`Series` vs `DataItem`). But I am not sure if we can rely on only checking the required properties or if we need to check the equality differently?

Reply:
`DataItem` doesn't have e.g. the property `series`. So I think we can't keep the same checks. I think the previous check of `entry.series === d.series` did only compile because the input was of type `any` (`@Input() activeEntries: any[];`). WDYT?

> Could you also fix the typing on the ngx-charts-series-horizontal component as well?

Aren't we already in `ngx-charts-series-horizontal`? Do you maybe mean `ngx-charts-series-vertical`? Or am I missing something 😇? Nevertheless, I updated it there as well, see 4acbd5339e1c34cddf6033e54b97b41ffdfacf49.

Reply:
Sorry I meant `ngx-charts-series-vertical` 
And in that case, let's go with what you have.
Thanks!